fvtest/compilertriltest/ArithmeticTest.cpp

```diff
@@ -34,6 +34,10 @@ int32_t imul(int32_t l, int32_t r) {
     return l*r;
 }
 
+int32_t imulh(int32_t l, int32_t r) {
+    return (int32_t)(((int64_t)l * (int64_t)r) >> 32);
```

Comment:
To make the code more readable, can you split it into multiple lines using temporary variables? Also, please use `static_cast` instead of c-style casts.